deploy/cert-renew.yml

```diff
@@ -6,6 +6,20 @@
         that:
           - cert_dir is defined
 
+    - name: 'Copy manual auth script'
+      copy:
+          src: '{{ item }}'
+          dest: '{{ dst_dir | mandatory }}/domain-auth/'
```

Comment:
`'{{ dst_dir | mandatory }}/domain-auth/'` is used a lot. It'd better be defined as a variable (let's say `domain_auth_dir`, in the inventory).

Reply:
This directory must be created before. Add the following task before this task:

```YAML
    - name: 'Create domain auth directory'
      file:
        path: '{{ domain_auth_dir | mandatory }}'
        state: directory
      become: yes
```